paymentsheet/src/main/java/com/stripe/android/paymentsheet/navigation/PaymentSheetScreen.kt

```diff
@@ -37,7 +37,7 @@ import java.io.Closeable
 import com.stripe.android.R as PaymentsCoreR
 
 internal val verticalModeBottomContentPadding = 20.dp
-internal val horizontalModeBottomContentPadding = 8.dp
+internal val horizontalModeBottomContentPadding = 20.dp
```

Comment:
if these have the same value, do we need to set this per-screen at all? Or can we just use 20.dp as bottom content padding in PaymentSheetScreen?